samples/grpc/local-drone-control-scala/src/main/resources/META-INF/native-image/generated/reflect-config.json

```diff
@@ -0,0 +1,5456 @@
+[
+  {
+    "name": "[B"
+  },
+  {
+    "name": "[C"
+  },
+  {
+    "name": "[D"
+  },
+  {
+    "name": "[F"
+  },
+  {
+    "name": "[I"
+  },
+  {
+    "name": "[J"
+  },
+  {
+    "name": "[Lakka.actor.ActorRef;"
+  },
+  {
+    "name": "[Lakka.actor.InternalActorRef;"
+  },
+  {
+    "name": "[Lakka.actor.LightArrayRevolverScheduler$TaskQueue;"
+  },
+  {
+    "name": "[Lakka.cluster.ddata.ORMultiMapKey;"
+  },
+  {
+    "name": "[Lakka.remote.artery.Association;"
+  },
+  {
+    "name": "[Lakka.stream.stage.GraphStageLogic;"
+  },
+  {
+    "name": "[Lcom.fasterxml.jackson.annotation.JsonCreator;"
+  },
+  {
+    "name": "[Lcom.fasterxml.jackson.databind.deser.Deserializers;"
+  },
+  {
+    "name": "[Lcom.fasterxml.jackson.databind.deser.KeyDeserializers;"
+  },
+  {
+    "name": "[Lcom.fasterxml.jackson.databind.deser.ValueInstantiators;"
+  },
+  {
+    "name": "[Lcom.fasterxml.jackson.databind.ser.Serializers;"
+  },
+  {
+    "name": "[Lcom.fasterxml.jackson.databind.type.TypeModifier;"
+  },
+  {
+    "name": "[Lcom.fasterxml.jackson.module.scala.introspect.PropertyDescriptor;"
+  },
+  {
+    "name": "[Lcom.google.protobuf.Descriptors$FileDescriptor;"
+  },
+  {
+    "name": "[Ljava.lang.Class;"
+  },
+  {
+    "name": "[Ljava.lang.String;"
+  },
+  {
+    "name": "[Ljava.lang.reflect.Constructor;"
+  },
+  {
+    "name": "[Ljavax.management.openmbean.CompositeData;"
+  },
+  {
+    "name": "[Lscala.Tuple2;"
+  },
+  {
+    "name": "[Lscala.Tuple3;"
+  },
+  {
+    "name": "[Lscala.Tuple4;"
+  },
+  {
+    "name": "[S"
+  },
+  {
+    "name": "[Z"
+  },
+  {
+    "name": "[[I"
+  },
+  {
+    "name": "akka.Done"
+  },
+  {
+    "name": "akka.NotUsed"
+  },
+  {
+    "name": "akka.actor.ActorCell",
+    "fields": [
+      {
+        "name": "akka$actor$dungeon$Children$$_childrenRefsDoNotCallMeDirectly"
+      },
+      {
+        "name": "akka$actor$dungeon$Children$$_functionRefsDoNotCallMeDirectly"
+      },
+      {
+        "name": "akka$actor$dungeon$Children$$_nextNameDoNotCallMeDirectly"
+      },
+      {
+        "name": "akka$actor$dungeon$Dispatch$$_mailboxDoNotCallMeDirectly"
+      }
+    ]
+  },
+  {
+    "name": "akka.actor.ActorIdentity"
+  },
+  {
+    "name": "akka.actor.ActorInitializationException"
+  },
+  {
+    "name": "akka.actor.ActorKilledException"
+  },
+  {
+    "name": "akka.actor.ActorRef"
+  },
+  {
+    "name": "akka.actor.ActorSelectionMessage"
+  },
+  {
+    "name": "akka.actor.Address"
+  },
+  {
+    "name": "akka.actor.Identify"
+  },
+  {
+    "name": "akka.actor.IllegalActorStateException"
+  },
+  {
+    "name": "akka.actor.InvalidActorNameException"
+  },
+  {
+    "name": "akka.actor.InvalidMessageException"
```

Comment:
Yes. The tracing agent records all reflective access. The serialization bindings are converted into a class -> serializer map, so they're all accessed reflectively. And it will record all access, whether or not it was actually found.